bcipy/helpers/language_model.py

```diff
@@ -63,9 +63,15 @@ def sym_appended(symbol_probs: List[Tuple[str, float]],
     """
     if sym_prob[0] in dict(symbol_probs):
         return symbol_probs
-    delta = sym_prob[1] / len(symbol_probs)
-    return [(sym, prob - delta) for sym, prob in symbol_probs] + [sym_prob]
 
+    symbols = [prob[0] for prob in symbol_probs]
+    probabilities = np.array([prob[1] for prob in symbol_probs])
+
+    all_probs = np.append(probabilities, sym_prob[1] / (1 - sym_prob[1]))
+    all_probs = all_probs / sum(all_probs)
```

Comment:
use a different variable name!

Reply:
I am not sure if I understood your concern?